investigacion/Investigación-de-usuarios.md

```diff
@@ -0,0 +1,25 @@
+# Investigación de usuarios #
+
+El objetivo de esta encuesta ha sido obtener información sobre el comportamiento de los usuarios de redes sociales respecto a contenido político.
+
+**Análisis**
+
+Para ello, se ha elaborado un cuestionario a través de Formularios de Google en el que se han recogido un total de 9 preguntas relacionadas con el contenido sobre política en redes sociales. La difusiòn se ha realizado a través de WhatsApp a amigos y conocidos de entre 19 y 26 años. Hemos logrado obtener 21 respuestas y, de entre los participantes, un 81% tenían entre 19 y 22 años, mientras que tan solo un 14,3% tenían entre 23 y 26 años.
+
+Ninguna persona encuestada de entre 23 y 26 años comparte contenido político en las redes sociales. Sin embargo, en las edades comprendidas entre 19 y 22 años, un 42,9% afirma haberlo hecho, mientras que una mayoría del 57,1% admite no haber disfundido este tipo de contenido.
+
+De todos los encuestados, un 88,9% comparte noticias de actualidad. Un 66,7% redistribuye publicaciones en redes y un 44,4% difunde memes sobre política.
```

Comment:
¿Cómo es posible que, si solamente un 42,9% de los que tienen 19-22 comparten contenido político, un 44% (del total, entiendo) difunda memes sobre política?

¿Podéis mejorar la redacción, o es que las preguntas estaban mal planteadas?